crates/driver/src/print_build.rs

```diff
@@ -853,13 +885,14 @@ mod tests {
     use crate::Commands::PrintBuild;
     use std::collections::BTreeMap;
 
-    fn example_opt(no_aggregate_source: bool) -> Opt {
+    fn example_opt(no_aggregate_source: bool, append: bool) -> Opt {
```

Comment:
similar to the boolean blindness.. What about an `enum WriteMode { Overwrite, Append }`